pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md

```diff
@@ -60,17 +60,17 @@ secret/preprovisioned-example-ssh-key labeled
 
 ## Create overrides
 
-If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you wish to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
+If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you want to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
 
 ```yaml
```

Comment:
```suggestion
```bash
```

Reply:
same here - 
true true - but - if you keep the yaml code bracket, the colors look cool in the UI ;)

Reply:
I like colors myself. :)